src/infrastructure/figma/figma-auth-service.ts

```diff
@@ -70,13 +76,30 @@ export class FigmaAuthService {
 			client_id: getConfig().figma.clientId,
 			redirect_uri: redirectUri,
 			scope: getConfig().figma.scope,
-			state: atlassianUserId, // TODO: MDTZ-1014: Use an anti-forgery state token to prevent Cross-Site Request Forgery (CSRF) attacks.
+			state: `${user.connectInstallationId}/${user.atlassianUserId}`, // TODO: MDTZ-1014: Use an anti-forgery state token to prevent Cross-Site Request Forgery (CSRF) attacks.
```

Comment:
**note:** This is a naive approach of bypassign `connectInstallationId` / `atlassianUserId` through the OAuth 2.0 flow. Hopefully, we can improve this part within MDTZ-1014 by encrypting (and, probably, signing the pair) with some private key.